scripts/armadillo-setup.sh

```diff
@@ -0,0 +1,298 @@
+#!/bin/bash
+
+ARMADILLO_SETUP_VER=1.0.1
+ARMADILLO_URL=https://github.com/molgenis/molgenis-service-armadillo/
+ARMADILLO_PROFILE=default
+ARMADILLO_PATH=/usr/share/armadillo
+ARMADILLO_CFG_PATH=/etc/armadillo
+ARMADILLO_SYS_USER=armadillo
+ARMADILLO_LOG_PATH=/var/log/armadillo
+ARMADILLO_AUDITLOG=$ARMADILLO_LOG_PATH/audit.log
+ARMADILLO_DATADIR=$ARMADILLO_PATH/data
+
+
+handle_args() {
+    while :
+    do
+        case "$1" in 
+        --version)
+          ARMADILLO_VERSION=$2
+          shift 2
+          ;;
```

Comment:
I think it's a good idea to add `--admin-email` here as well. This will allow the admin to immediately login with OIDC instead of first having to login with basic auth, then add the email, and reauthenticate.

Reply:
See `oidc-admin-user` in the config